calculus/derivative-rules.md

```diff
@@ -1 +1,66 @@
 # Product, quotient, and chain rule
+
+## Product Rule
+If $f(x) = u(x)  v(x)$ then:
+
+$$f' = uv' + vu'$$
+
+"u-dv plus v-du"
+"first-derivative-of-second plus second-derivative-of-first"
+
+### Example 1:
+$f(x) = (x-3)(x-1)$
+
+**Old way:** Expand The Polynomial 
+<br>
+$f(x) = x^2 - 4x + 3$  
+$f'(x) = 2x - 4$
+
+**New way:** 
+<br>
+$u = x-3$ and  $v = x-1$
+<br>
+$u' = 1$ and $v' = 1$
+  
+ $f'(x) =  \underbrace{(x-3)}_u\underbrace{(1)}_{v'}+ \underbrace{(x-1)}_v\underbrace{(1)}_{u'}= 2x - 4$
```

Comment:
Please make use of `{align}` to group multiple equations together. It also isn't necessary to exactly reproduce all the marks in my notes, so I simplified your final result. Last, please make sure there are no stray HTML tags like `<br>`.
```suggestion
**Old way:** Expand The Polynomial 

\begin{align}
f(x) &= x^2 - 4x + 3 \\
f'(x) &= 2x - 4
\end{align}

**New way:** 

\begin{align
u &= x-3 &  v &= x-1 \\
u' &= 1 & v' &= 1
\end{align}
  
 so
 
 \begin{equation}
 f'(x) =  (x-3) \cdot 1 + (x-1) \cdot 1 = 2x - 4
 \end{equation
```